From 426835f3cf26be344eaf6ac978c58ef3ffe9f911 Mon Sep 17 00:00:00 2001
From: John Jacques <john.jacques@intel.com>
Date: Wed, 7 Oct 2015 14:02:11 -0500
Subject: [PATCH 100/278] arch/arm64: Axxia Device Tree Update for XLF

As there are more than 16 cores, more memory is needed for
the redistributor registers.

Signed-off-by: John Jacques <john.jacques@intel.com>
---
 arch/arm64/boot/dts/intel/axc67xx.dtsi | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm64/boot/dts/intel/axc67xx.dtsi b/arch/arm64/boot/dts/intel/axc67xx.dtsi
index 6fb6b9e..fd6e42d 100644
--- a/arch/arm64/boot/dts/intel/axc67xx.dtsi
+++ b/arch/arm64/boot/dts/intel/axc67xx.dtsi
@@ -68,7 +68,7 @@
 		#address-cells = <0>;
 		interrupt-controller;
 		reg = <0x80 0x10000000 0x10000>,
-		      <0x80 0x10400000 0x200000>;
+		      <0x80 0x10400000 0x400000>;
 		interrupts = <GIC_PPI 9 IRQ_TYPE_LEVEL_HIGH>;
 	};
 
-- 
1.8.5.2.233.g932f7e4

